_sass/all.scss

```diff
@@ -31,7 +32,7 @@
 
 body {
     background-color: #f5f5f9;
-    margin-bottom: 50px;
+    // margin-bottom: 50px;
```

Comment:
Always remember to remove commented code before pushing your changes. Can you please do that?

Reply:
> Always remember to remove commented code before pushing your changes. Can you please do that?

fixed!